### ISSUE TITLE
Code Crashed While operating shooterTestStable

### ISSUE THREAD

**Issue by AndrewIverson** [AusTINCANsProgrammingTeam/2022RobotCode#71]
```
Unhandled exception: java.lang.NullPointerException
   from: frc.robot.subsystems.ShooterSubsystem.adjustHood(ShooterSubsystem.java:50)
     at: at frc.robot.subsystems.ShooterSubsystem.adjustHood(ShooterSubsystem.java:50)
     at: at frc.robot.subsystems.ShooterSubsystem.prime(ShooterSubsystem.java:142)
     at: at frc.robot.commands.ShooterPrime.initialize(ShooterPrime.java:33)
     at: at edu.wpi.first.wpilibj2.command.CommandScheduler.initCommand(CommandScheduler.java:141)
     at: at edu.wpi.first.wpilibj2.command.CommandScheduler.schedule(CommandScheduler.java:184)
     at: at edu.wpi.first.wpilibj2.command.CommandScheduler.schedule(CommandScheduler.java:214)
     at: at edu.wpi.first.wpilibj2.command.Command.schedule(Command.java:259)
     at: at edu.wpi.first.wpilibj2.command.button.Trigger$2.run(Trigger.java:151)
     at: at edu.wpi.first.wpilibj2.command.CommandScheduler.run(CommandScheduler.java:259)
     at: at frc.robot.Robot.robotPeriodic(Robot.java:51)
     at: at edu.wpi.first.wpilibj.IterativeRobotBase.loopFunc(IterativeRobotBase.java:328)
     at: at edu.wpi.first.wpilibj.TimedRobot.startCompetition(TimedRobot.java:131)
     at: at edu.wpi.first.wpilibj.RobotBase.runRobot(RobotBase.java:373)
     at: at edu.wpi.first.wpilibj.RobotBase.startRobot(RobotBase.java:463)
     at: at frc.robot.Main.main(Main.java:22)

Error at frc.robot.subsystems.ShooterSubsystem.adjustHood(ShooterSubsystem.java:50): Unhandled exception: java.lang.NullPointerException
The robot program quit unexpectedly. This is usually due to a code error.
The above stacktrace can help determine where the error occurred.
See https://wpilib.org/stacktrace for more information.
at frc.robot.subsystems.ShooterSubsystem.adjustHood(ShooterSubsystem.java:50)
at frc.robot.subsystems.ShooterSubsystem.prime(ShooterSubsystem.java:142)
at frc.robot.commands.ShooterPrime.initialize(ShooterPrime.java:33)
at edu.wpi.first.wpilibj2.command.CommandScheduler.initCommand(CommandScheduler.java:141)
at edu.wpi.first.wpilibj2.command.CommandScheduler.schedule(CommandScheduler.java:184)
at edu.wpi.first.wpilibj2.command.CommandScheduler.schedule(CommandScheduler.java:214)
at edu.wpi.first.wpilibj2.command.Command.schedule(Command.java:259)
at edu.wpi.first.wpilibj2.command.button.Trigger$2.run(Trigger.java:151)
at edu.wpi.first.wpilibj2.command.CommandScheduler.run(CommandScheduler.java:259)
at frc.robot.Robot.robotPeriodic(Robot.java:51)
at edu.wpi.first.wpilibj.IterativeRobotBase.loopFunc(IterativeRobotBase.java:328)
at edu.wpi.first.wpilibj.TimedRobot.startCompetition(TimedRobot.java:131)
at edu.wpi.first.wpilibj.RobotBase.runRobot(RobotBase.java:373)
at edu.wpi.first.wpilibj.RobotBase.startRobot(RobotBase.java:463)
at frc.robot.Main.main(Main.java:22)
Warning at edu.wpi.first.wpilibj.RobotBase.runRobot(RobotBase.java:388): The robot program quit unexpectedly. This is usually due to a code error.
The startCompetition() method (or methods called by it) should have handled the exception above.
The above stacktrace can help determine where the error occurred.
See https://wpilib.org/stacktrace for more information.
Error at edu.wpi.first.wpilibj.RobotBase.runRobot(RobotBase.java:395): The startCompetition() method (or methods called by it) should have handled the exception above.
#
# A fatal error has been detected by the Java Runtime Environment:
#
# SIGSEGV (0xb) at pc=0xb6de33d8, pid=1889, tid=1895
#
# JRE version: OpenJDK Runtime Environment (11.0.12.5) (build 11.0.12.5-frc+0-2022-11.0.12u5-1)
# Java VM: OpenJDK Client VM (11.0.12.5-frc+0-2022-11.0.12u5-1, mixed mode, concurrent mark sweep gc, linux-)
# Problematic frame:
# C [libc.so.6+0x6c3d8]
#
# No core dump will be written. Core dumps have been disabled. To enable core dumping, try "ulimit -c unlimited" before starting Java again
#
# An error report file with more information is saved as:
# /tmp/hs_err_pid1889.log
pure virtual method called
terminate called without an active exception
********** Robot program starting **********
NT: server: client CONNECTED: 10.21.58.184 port 53408
```

**Comment by AndrewIverson** [comment]
```
Warning at edu.wpi.first.wpilibj.Tracer.lambda$printEpochs$0(Tracer.java:63): SmartDashboard.updateValues(): 0.000153s
robotPeriodic(): 0.002881s
LiveWindow.updateValues(): 0.001692s
Shuffleboard.update(): 0.017730s
disabledPeriodic(): 0.000137s
NT: server: client CONNECTED: 10.21.58.184 port 58220
Unhandled exception: java.lang.NullPointerException
Error at frc.robot.subsystems.ShooterSubsystem.adjustHood(ShooterSubsystem.java:50): Unhandled exception: java.lang.NullPointerException
The robot program quit unexpectedly. This is usually due to a code error.
The above stacktrace can help determine where the error occurred.
See https://wpilib.org/stacktrace for more information.
at frc.robot.subsystems.ShooterSubsystem.adjustHood(ShooterSubsystem.java:50)
at frc.robot.subsystems.ShooterSubsystem.prime(ShooterSubsystem.java:137)
at frc.robot.commands.ShooterPrime.initialize(ShooterPrime.java:33)
at edu.wpi.first.wpilibj2.command.CommandScheduler.initCommand(CommandScheduler.java:141)
at edu.wpi.first.wpilibj2.command.CommandScheduler.schedule(CommandScheduler.java:184)
at edu.wpi.first.wpilibj2.command.CommandScheduler.schedule(CommandScheduler.java:214)
at edu.wpi.first.wpilibj2.command.Command.schedule(Command.java:259)
at edu.wpi.first.wpilibj2.command.button.Trigger$2.run(Trigger.java:151)
at edu.wpi.first.wpilibj2.command.CommandScheduler.run(CommandScheduler.java:259)
at frc.robot.Robot.robotPeriodic(Robot.java:51)
at edu.wpi.first.wpilibj.IterativeRobotBase.loopFunc(IterativeRobotBase.java:328)
at edu.wpi.first.wpilibj.TimedRobot.startCompetition(TimedRobot.java:131)
at edu.wpi.first.wpilibj.RobotBase.runRobot(RobotBase.java:373)
at edu.wpi.first.wpilibj.RobotBase.startRobot(RobotBase.java:463)
at frc.robot.Main.main(Main.java:22)
Warning at edu.wpi.first.wpilibj.RobotBase.runRobot(RobotBase.java:388): The robot program quit unexpectedly. This is usually due to a code error.
The above stacktrace can help determine where the error occurred.
See https://wpilib.org/stacktrace for more information.
The startCompetition() method (or methods called by it) should have handled the exception above.
Error at edu.wpi.first.wpilibj.RobotBase.runRobot(RobotBase.java:395): The startCompetition() method (or methods called by it) should have handled the exception above.
#
# A fatal error has been detected by the Java Runtime Environment:
#
# SIGSEGV (0xb) at pc=0xb6da03d8, pid=1712, tid=1718
#
# JRE version: OpenJDK Runtime Environment (11.0.12.5) (build 11.0.12.5-frc+0-2022-11.0.12u5-1)
# Java VM: OpenJDK Client VM (11.0.12.5-frc+0-2022-11.0.12u5-1, mixed mode, concurrent mark sweep gc, linux-)
# Problematic frame:
# C [libc.so.6+0x6c3d8]
#
# No core dump will be written. Core dumps have been disabled. To enable core dumping, try "ulimit -c unlimited" before starting Java again
#
# An error report file with more information is saved as:
# /tmp/hs_err_pid1712.log
pure virtual method called
terminate called without an active exception
********** Robot program starting **********
NT: server: client CONNECTED: 10.21.58.184 port 61265
Drivebase enabled
CDS enabled
Intake enabled
Shooter enabled
java.io.IOException: Cannot open file: /home/lvuser/deploy/Straight.wpilib.json
at edu.wpi.first.math.WPIMathJNI.fromPathweaverJson(Native Method)
at edu.wpi.first.math.trajectory.TrajectoryUtil.fromPathweaverJson(TrajectoryUtil.java:78)
at frc.robot.RobotContainer.initializeTrajectories(RobotContainer.java:218)
at frc.robot.RobotContainer.<init>(RobotContainer.java:166)
at frc.robot.Robot.robotInit(Robot.java:31)
at edu.wpi.first.wpilibj.TimedRobot.startCompetition(TimedRobot.java:107)
at edu.wpi.first.wpilibj.RobotBase.runRobot(RobotBase.java:373)
at edu.wpi.first.wpilibj.RobotBase.startRobot(RobotBase.java:463)
at frc.robot.Main.main(Main.java:22)
********** Robot program startup complete **********
```

It did it again after disabling and reenabling.

**Comment by AndrewIverson** [comment]
Fixed with commit 790f77b8b23c3058c90da64fc5d9ab0eb1fb734c
